### PR TITLE
add description to group argument at match regex script

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

**Comment by meirwah** [comment]
@mayairv can you review this desc change?

**Comment by meirwah** [comment]
still waiting @mayairv ...

**Comment by meirwah** [comment]
Or @slavikm plz review

File: Scripts/script-MatchRegex.yml [hunk 1]
```diff
@@ -35,9 +35,10 @@ args:
   description: The regex to match and extract
 - name: group
   description: The matching group to return (if not provided will return the full
```
Comment (mayairv) [inline Scripts/script-MatchRegex.yml]: (if not provided the full match will be returned).

File: Scripts/script-MatchRegex.yml [hunk 2]
```diff
@@ -35,9 +35,10 @@ args:
...
-    match)
+    match), group value should start at 1
 - name: contextKey
   description: The context key to populate with the result
 - name: flags
   description: The regex flags for the match - i is the default
```
Comment (mayairv) [inline Scripts/script-MatchRegex.yml]: The regex flags to match. Default is - i.

**Comment by meirwah** [comment]
@slavikm can you approve?

**Pull request merged**